Improve actionability of `Property 'formatting' is misspelled or does not exist.`

<!-- Your feature request may already be reported! -->
<!-- Please search in the [issues](https://github.com/detekt/detekt/issues) before creating a new one. -->

## Expected Behavior
<!--- If you're suggesting a change/improvement, tell us how it should work -->
I'm merely suggesting to tweak the error message so that it hints the issue is not at all with formatting but that a _plugin_ is missing.

E.g. 

```diff
- Property 'formatting' is misspelled or does not exist.
+ Property 'formatting' is misspelled or does not exist. This error may also indicate a detekt plugin is necessary to handle the 'formatting' key.
```

## Current Behavior
<!--- If suggesting a change/improvement, explain the difference from the current behavior -->
If you happen to copy a config from a project and you're new to detekt. You may not be aware that detekt has plugins.

If such config has for example the `formatting` entry in the yaml file, when running the detekt task the user will be prompted with a failure.

```
> Run failed with 1 invalid config property.
        - Property 'formatting' is misspelled or does not exist.
```

## Context
<!--- What are you trying to accomplish? -->
<!--- Providing context helps us come up with a solution that is most useful in the real world -->

This happened to me, and it took me longer that I should have to realise the setup was missing a detekt plugin. I was stuck until I saw this https://github.com/detekt/detekt/issues/5960#issuecomment-1494667125 (issue is closed).

And indeed the fix is quick and simple, i.e. adding the _formatting_ plugin.

.`build.gradle.kts`
```diff
+ val detektPlugins: Configuration = configurations.getByName("detektPlugins")
+ 
+ dependencies {
+     detektPlugins(libs.bundles.detektplugins)
+ }
```

.`libs.versions.toml`
```diff
  [versions]
  detekt = "1.23.1"

  [libraries]
+ detekt-formatting = { module = "io.gitlab.arturbosch.detekt:detekt-formatting", version.ref = "detekt" }

  [bundles]
+ detektplugins = ["detekt-formatting"]
```

Not sure you can cess the build scan but just in case : https://scans.gradle.com/s/7r6tqpxq3fxua

* gradle `8.3`
* detekt `1.23.1`

<details><summary>Stack trace exerpt</summary>

```text
<html><head></head><body>

Caused by: org.gradle.api.GradleException: [31mRun failed with 1 invalid config property.[0m 
	- [31mProperty 'formatting' is misspelled or does not exist.[0m</body></html>
	at io.gitlab.arturbosch.detekt.invoke.DefaultCliInvoker.invokeCli(DetektInvoker.kt:102)
...
Caused by: io.github.detekt.tooling.api.InvalidConfig: [31mRun failed with 1 invalid config property.[0m
	- [31mProperty 'formatting' is misspelled or does not exist.[0m</body></html>
	at io.gitlab.arturbosch.detekt.core.config.validation.ConfigValidationKt.checkConfiguration(ConfigValidation.kt:53)
```

</details>

--- a/detekt-core/src/main/kotlin/io/gitlab/arturbosch/detekt/core/config/validation/InvalidPropertiesConfigValidator.kt
+++ b/detekt-core/src/main/kotlin/io/gitlab/arturbosch/detekt/core/config/validation/InvalidPropertiesConfigValidator.kt
@@ -79,7 +79,7 @@ internal class InvalidPropertiesConfigValidator(
 
         internal fun propertyDoesNotExists(prop: String): Notification =
             SimpleNotification(
-                "Property '$prop' is misspelled or does not exist." +
+                "Property '$prop' is misspelled or does not exist. " +
                     "This error may also indicate a detekt plugin is necessary to handle the '$prop' key."
             )
 

--- a/detekt-core/src/main/kotlin/io/gitlab/arturbosch/detekt/core/config/validation/InvalidPropertiesConfigValidator.kt
+++ b/detekt-core/src/main/kotlin/io/gitlab/arturbosch/detekt/core/config/validation/InvalidPropertiesConfigValidator.kt
@@ -78,7 +78,10 @@ internal class InvalidPropertiesConfigValidator(
     companion object {
 
         internal fun propertyDoesNotExists(prop: String): Notification =
-            SimpleNotification("Property '$prop' is misspelled or does not exist.")
+            SimpleNotification(
+                "Property '$prop' is misspelled or does not exist." +
+                    "This error may also indicate a detekt plugin is necessary to handle the '$prop' key."
+            )
 
         internal fun nestedConfigurationExpected(prop: String): Notification =
             SimpleNotification("Nested config expected for '$prop'.")

